test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -0,0 +1,129 @@
+﻿function Write-LogPassThru
+{
+    Param(
+        [Parameter(Mandatory=$true, ValueFromPipeline=$true, Position = 0)]
+        [string] $Message,
+        $Path = "$env:Temp\CodeCoverageRunLogs.txt"
+    )
+
+    $message = "$([Datetime]::Now.ToShortDateString()) - $([datetime]::Now.ToShortTimeString()) : $message"
+
+    if(-not (Test-Path $Path))
+    {
+        Set-Content -Path $Path -Value $Message -Force -PassThru
+    }
+    else
+    {
+        Add-Content -Path $Path -Value $Message -PassThru
+    }
+}
+
+$codeCoverageZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/CodeCoverage.zip'
+$testContentZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/tests.zip'
+$openCoverZip = 'https://ci.appveyor.com/api/projects/PowerShell/powershell-f975h/artifacts/OpenCover.zip'
+
+Write-LogPassThru -Message "codeCoverageZip: $codeCoverageZip"
+Write-LogPassThru -Message "testcontentZip: $testContentZip"
+Write-LogPassThru -Message "openCoverZip: $openCoverZip"
+
+$outputBaseFolder = "$env:Temp\CC"
+$null = New-Item -ItemType Directory -Path $outputBaseFolder -Force
```

Comment:
this path may already exist with contents. Would it be a good thing to delete the directory (recursively) if it exists before starting?